### PR TITLE
Add traits for Tian Xia World Guide

### PR DESCRIPTION
**Pull request by TMunsie**

### CODE REVIEW

File: packs/vehicles/karakoa.json [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+                    "value": ""
+                },
+                "publication": {
+                    "license": "OGL",
+                    "remaster": false,
+                    "title": ""
+                },
+                "rules": [
+                    {
+                        "key": "Resistance",
```
Comment (simonward) [inline packs/vehicles/karakoa.json:28]: For other vehicles, we've used:
```
{
	"key": "Weakness",
	"predicate": [
		{
			"not": "self:condition:broken"
		}
	],
	"type": "fire",
	"value": 10
}
```

File: packs/spells/power-of-the-beasts.json [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
...kles\"></i>Atone</a> ritual before attempting this ritual again.</p>\n<hr />\n<p><strong>Critical Success</strong> As success, except you can defeat three enemies that are only one size larger than you instead.</p>\n<p><strong>Success</strong> The power begins to take root. Once you've defeated an enemy two or more sizes larger than you in single combat, you gain the <a class=\"content-link\" draggable=\"true\" data-uuid=\"Compendium.pf2e.feats-srd.Item.KxaYlC50zzHysJj8\" data-id=\"KxaYlC50zzHysJj8\" data-type=\"Item\" data-pack=\"pf2e.feats-srd\" data-tooltip=\"Feat/Feature Item\"><i class=\"fa-solid fa-medal\"></i>Titan Wrestler</a> feat.</p>\n<p><strong>Failure</strong> You drink the soup or wine, but the understanding eludes you.</p>\n<p><strong>Critical Failure</strong> The drink contains shards of bone you didn't grind down properly. You're <a class=\"content-link\" draggable=\"true\" data-uuid=\"Compendium.pf2e.conditionitems.Item.fesd1n5eVhpCSS18\" data-id=\"fesd1n5eVhpCSS18\" data-type=\"Item\" data-pack=\"pf2e.conditionitems\" data-tooltip=\"Condition Item\"><i class=\"fa-solid fa-face-zany\"></i>Sickened 2</a> for 1 week, during which the condition can't be reduced.</p>"
```
Comment (CarlosFdez) [inline packs/spells/power-of-the-beasts.json:14]: Seems your links got a bit weird here

File: packs/vehicles/karakoa.json (hunk 1 above)
Comment (TMunsie) [reply packs/vehicles/karakoa.json:28]: The broken condition isn't added at BT for vehicles. This is what I initially tried but it doesn't work.

File: packs/spells/power-of-the-beasts.json (hunk 2 above)
Comment (TMunsie) [reply packs/spells/power-of-the-beasts.json]: Fixed

**Pull request merged**